metadium/admin.go

```diff
@@ -1084,12 +1086,22 @@ func LogBlock(height int64, hash common.Hash) {
 	}
 
 	tstart := time.Now()
-	if err := admin.etcdPut("metadium-work", string(work)); err != nil {
+	rev, err := admin.etcdPut("metadium-work", string(work))
+	if err != nil {
 		log.Error("Metadium - failed to log the latest block",
 			"height", height, "hash", hash, "took", time.Since(tstart))
 	} else {
 		log.Info("Metadium - logged the latest block",
 			"height", height, "hash", hash, "took", time.Since(tstart))
+
+		if (rev%etcdCompactFrequency == 0) && (rev > 100) {
```

Comment:
Shouldn't "100" be replaced by etcdCompactFrequency? I.e.

if rev%etcdCompactFreqency == 0 && rev > etcdCompactFreqency {

Reply:
Thanks! I added distinct var, named `etcdCompactWindow`, for more generalization and flexibility.
`etcdCompactWindow` is set to 100 as default.

https://github.com/lukepark327/go-metadium/commit/78e7ad9599fcfd95c276d89157ded8fb7e3a5ad2

Reply:
And, I updated codes to apply concept of compact window. Please check the following commits: https://github.com/METADIUM/go-metadium/pull/18/commits/73062f1f1a832022380d83f89de1a834f0c30523